tools/pipelines/build-docs.yml

```diff
@@ -2,7 +2,14 @@
 # Licensed under the MIT License.
 
 # build-docs pipeline
-# This pipeline builds the main branch docs each time the client packages are built
+# This pipeline builds the main branch docs as well as previous major release branch docs each time the client packages are built.
```

Comment:
Nit: I would express these docs in terms of the "1.0" branch, rather than the "lts" branch. LTS will change over time, whereas it's the 1.0 branch specifically with shortcomings.

But otherwise, these docs look great! Thank you for the thorough explanation! This will make future devs' lives much easier when working in here.